scripts/install_cmake.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
I don't know if it is important, but better use
~~~
#!/usr/bin/env bash
~~~
?

Reply:
Doesn't matter really. Afaik, all Linuxes have their bash at `/bin/bash`. This might not me the case with OSX or some BSD. But a) this script is currently only for Travis, running Ubuntu, and b) if you want to compile NuTo on other OSs, I guess this would be your smallest problem.